packages/cozy-intent/package.json

```diff
@@ -15,9 +15,12 @@
   },
   "devDependencies": {
     "@babel/core": "7.12.3",
+    "@testing-library/react": "10.4.9",
     "babel-preset-cozy-app": "^2.0.1",
     "eslint-config-cozy-app": "^4.0.0",
-    "react": "17.0.2",
+    "jest": "26.2.2",
+    "mutationobserver-shim": "^0.3.7",
```

Comment:
let's remove that 🙏

Reply:
I forgot to change the PR target after a rebase. This comment is related to : https://github.com/cozy/cozy-libs/pull/1438